tests/unit/conftest.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Unit test configuration."""
+import ops.testing
+import pytest
+
+from charm import IdentityPlatformLoginUiOperatorCharm
+
+ops.testing.SIMULATE_CAN_CONNECT = True
+
+
+@pytest.fixture()
+def harness(mocked_kubernetes_service_patcher):
+    """Initialize harness with Charm."""
+    harness = ops.testing.Harness(IdentityPlatformLoginUiOperatorCharm)
+    harness.set_model_name("testing")
+    harness.set_leader(True)
```

Comment:
I'd recommend not setting the leader in the fixture and be specific about it in tests. It sort of assumes that the unit will always be the leader which is obviously not always the case.

Reply:
done